arch/cortex-m0/src/lib.rs

```diff
@@ -111,7 +111,7 @@ MEXC_RETURN_MSP:
   .word 0xFFFFFFF9
 MEXC_RETURN_PSP:
   .word 0xFFFFFFFD"
-    : : : : "volatile");
+    );
```

Comment:
```suggestion
    , options(noreturn)
    );
```
the assembly manually jumps out. Also, naked functions require a single `noreturn` inline asm block -- it is UB for an inline asm block inside a naked function to return.

This suggestion will probably need formatting